packages/runed/vite.config.ts

```diff
@@ -26,8 +26,8 @@ export default defineConfig({
 				extends: true,
 				test: {
 					setupFiles: ["./setupTest.ts"],
-					include: ["src/**/*.{test,test.svelte,spec}.{js,ts}"],
-					exclude: ["src/**/*.browser.{test,test.svelte,spec}.{js,ts}"],
+					include: ["src/**/*.{test,test.svelte}.{js,ts}"],
+					exclude: ["src/**/*.browser.{test,test.svelte}.{js,ts}"],
```

Comment:
To separate testing files, vitest now only finds `test.{js,ts}`, and playwright finds `spec.{js,ts}`.